python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -406,9 +406,6 @@ def isA(self, _what):
 
         what = stripProxy(_what)
 
-        if isinstance(what, types.InstanceType):
```

Comment:
Is this a problem removing this? I'm not sure I understand what this does anyway...